code/game/objects/structures/lavaland/necropolis_tendril.dm

```diff
@@ -49,7 +49,6 @@ GLOBAL_LIST_INIT(tendrils, list())
 	return TRUE
 
 /obj/structure/spawner/lavaland/deconstruct(disassembled)
-	new /obj/effect/collapse(loc)
```

Comment:
wait why are you removing the collapse?

Reply:
At the original PR comment:

> Removes chasms appearing around destroyed tendrils, as compensation.

The `chasm` here is caused by `/obj/effect/collapse`. So it seems that the author tries to give miners some form of a compensation for adding a special lock.

Reply:
that is stupid

Reply:
okay, readded